test/WormholeL1VotePool.t.sol

```diff
@@ -259,7 +292,81 @@ contract _ReceiveCastVoteWormholeMessages is L1VotePoolTest {
       new bytes[](0),
       bytes32(uint256(uint160(address(l2VoteAggregator)))),
       L2_CHAIN.wormholeChainId,
-      bytes32("")
+      bytes32(""),
+      false
+    );
+  }
+
+  function testFuzz_RevertWhen_VoteBeforeProposalStart(
+    uint32 _l2Against,
+    uint32 _l2InFavor,
+    uint32 _l2Abstain,
+    uint32 _l2NewAgainst,
+    uint32 _l2NewInFavor,
+    uint32 _l2NewAbstain
+  ) public {
+    vm.assume(_l2NewAgainst < _l2Against);
+    vm.assume(_l2NewInFavor < _l2InFavor);
+    vm.assume(_l2NewAbstain < _l2Abstain);
+
+    vm.selectFork(targetFork);
+
+    l1Erc20.approve(
+      address(l1VotePool), uint96(_l2Against) + uint96(_l2InFavor) + uint96(_l2Abstain)
+    );
+    l1Erc20.mint(address(this), uint96(_l2Against) + uint96(_l2InFavor) + uint96(_l2Abstain));
+    l1Erc20.delegate(address(l1VotePool));
+
+    uint256 _proposalId = l1VotePool.createProposalVote(address(l1Erc20));
+    console2.logString("vote created");
+
+    vm.prank(L1_CHAIN.wormholeRelayer);
+    vm.expectRevert("Governor: vote not currently active");
+    l1VotePool.receiveWormholeMessages(
+      abi.encode(_proposalId, _l2NewAgainst, _l2NewInFavor, _l2NewAbstain),
+      new bytes[](0),
+      bytes32(uint256(uint160(address(l2VoteAggregator)))),
+      L2_CHAIN.wormholeChainId,
+      bytes32(""),
+      false
+    );
+  }
+
+  function testFuzz_RevertWhen_VoteAfterProposalEnd(
+    uint32 _l2Against,
+    uint32 _l2InFavor,
+    uint32 _l2Abstain,
+    uint32 _l2NewAgainst,
+    uint32 _l2NewInFavor,
+    uint32 _l2NewAbstain
+  ) public {
+    vm.assume(_l2NewAgainst < _l2Against);
+    vm.assume(_l2NewInFavor < _l2InFavor);
+    vm.assume(_l2NewAbstain < _l2Abstain);
+
+    vm.selectFork(targetFork);
+
+    l1Erc20.approve(
+      address(l1VotePool), uint96(_l2Against) + uint96(_l2InFavor) + uint96(_l2Abstain)
+    );
+    l1Erc20.mint(address(this), uint96(_l2Against) + uint96(_l2InFavor) + uint96(_l2Abstain));
+    l1Erc20.delegate(address(l1VotePool));
+
+    console2.logString("Proposal block");
+    console2.logUint(block.number);
+    uint256 _proposalId = l1VotePool.createProposalVote(address(l1Erc20));
+    l1VotePool._jumpToProposalEnd(_proposalId, 1);
+    console2.logString("vote created");
+
+    vm.prank(L1_CHAIN.wormholeRelayer);
+    vm.expectRevert("Governor: vote not currently active");
```

Comment:
Maybe my logic in the prior comment is wrong, because now we're testing `RevertWhen_VoteAfterProposalEnd` and there's still no roll?

Reply:
We have this jump to end function call here that does the roll. https://github.com/ScopeLift/l2-flexible-voting/pull/42/files#diff-abb0e5a0d17cbc8fc275bb18828b0b914daa8189b1f40d4df29f7dbdb4dc486aR355